### DISCUSSION TITLE
[Feature request.] System tray icons in multiple monitors

### DISCUSSION THREAD

**Discussion by JakubovBrloh** [valinet/ExplorerPatcher#1915] [Ideas]
I know this is kinda obvious.
But for me and maybe for some of people this feature would be VERY appealing.

This is what I have - tray icons are only in/on the primary monitor. But the secondary monitor just have clock-date.
![mon icons1](https://github.com/valinet/ExplorerPatcher/assets/110103493/7c255a51-385f-46dc-a6f2-815b713b3967)
![mon icons2](https://github.com/valinet/ExplorerPatcher/assets/110103493/bd2710dd-e0d6-415e-a959-98b6f93d6881)
.

> The (edited) image below is what I imagine:

![mon icons2_2](https://github.com/valinet/ExplorerPatcher/assets/110103493/a78c333c-36b2-4af1-883c-31fc4634d68d)

**Comment by pyrates999** [comment]
I don't think the system tray icons were ever visible on any monitor but the primary monitor.

**Reply by S1332** [reply]
but i used to be able to pull the main system bar on the offscreen so i could monitor things in my tray icons without needing to get out of fullscreen games. really hope there is a way in the future to do that, i've just upgraded and did not expect win 11 to be THAT bad

**Comment by Exthreal3** [comment]
Been looking for this also, plus the option to re-locate the system tray (Left or Right) / Start button

**Comment by pyrates999** [comment]
Discussed here: https://github.com/valinet/ExplorerPatcher/discussions/578
